tests/api/mam_test.c

```diff
@@ -136,6 +136,56 @@ void test_write_until_next_channel(void) {
   }
 }
 
+void test_encrypt_decrypt_psk(void) {
+  const char* json_template_send =
+      "{\"x-api-key\":\"" TEST_TOKEN
+      "\",\"data\":{\"seed\":\"%s\",\"ch_mss_depth\":" STR(TEST_CH_DEPTH) ",\"message\":\"" TEST_PAYLOAD
+                                                                          "\"},\"key\":{\"psk\":[\"" TEST_PSK
+                                                                          "\"]}, \"protocol\":\"MAM_V1\"}";
+  char seed[NUM_TRYTES_ADDRESS + 1] = {};
+  gen_rand_trytes(NUM_TRYTES_ADDRESS, (tryte_t*)seed);
+  const int len_send = strlen(json_template_send) + NUM_TRYTES_ADDRESS;
```

Comment:
The len_send length should be strlen(json_template_send) + NUM_TRYTES_ADDRESS + 1 to avoid snprintf buffer overflow.

Reply:
I think use size_t to instead of int would be better.